centreon/www/install/php/Update-next.tpl.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/*
+ * Copyright 2005 - 2025 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+require_once __DIR__ . '/../../../bootstrap.php';
+
+/**
+ * This file contains changes to be included in the next version.
+ * The actual version number should be added in the variable $version.
+ */
+$version = '';
+$errorMessage = '';
+
+//----------------------
+// FUNCTIONS DEFINITIONS
+//----------------------
+
+try {
+    //----------------------
+    // FUNCTIONS EXECUTION
+    //----------------------
+
+    // DDL statements for real time database
+
+    // DDL statements for configuration database
+
+    // Transactional queries
+    if (! $pearDB->inTransaction()) {
+        $pearDB->beginTransaction();
+    }
+
+    $pearDB->commit();
```

Comment:
Use TODO instead of comments :) 
```suggestion
    // DDL statements for real time database
    // TODO add your function calls to update the real time database structure here

    // DDL statements for configuration database
    // TODO add your function calls to update the configuration database structure here

    // Transactional queries for configuration database
    if (! $pearDB->inTransaction()) {
        $pearDB->beginTransaction();
    }
    
    // TODO add your function calls to update the configuration database data here

    $pearDB->commit();
```

Reply:
ok